addons/mass_mailing/views/assets.xml

```diff
@@ -22,7 +22,7 @@
             * {
                 box-sizing: border-box !important;
             }
-            * h1, h2, h3, h4, h5, h6, p, td, th, div {
+            .o_layout :not(.fa) {
```

Comment:
Isn't that going to affect way more nodes than originally intended ?

Reply:
Well _everything_ but font awesome should be in that same font anyway, I don't really know why we limited it to these specific elements (I believe I generalized it in the perf branch already - anticipating conflict here ^^). But it absolutely should be limited to the mail.